### PR TITLE
feat: public user profile pic

### PR DESCRIPTION
**Pull request by Yash-pede**

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-source .env
+# source .env
```
Comment (dhirenmathur) [inline start.sh]:
```suggestion
source .env
```

File: app/modules/users/user_service.py [hunk 2]
```diff
@@ -143,3 +146,14 @@ def get_user_ids_by_emails(self, emails: List[str]) -> List[str]:
         except Exception as e:
             logger.error(f"Error fetching user ID by emails {emails}: {e}")
             return None
+    def get_user_profile_pic(self, uid: str) -> UserProfileResponse:
```
Comment (dhirenmathur) [inline app/modules/users/user_service.py]:
```suggestion


    def get_user_profile_pic(self, uid: str) -> UserProfileResponse:

Comment (dhirenmathur) [reply app/modules/users/user_service.py]: make this function and the internal api call async @Yash-pede

**Review by dhirenmathur — DISMISSED** [review]

File: app/modules/users/user_service.py [hunk 3]
```diff
@@ -143,3 +147,16 @@ def get_user_ids_by_emails(self, emails: List[str]) -> List[str]:
         except Exception as e:
             logger.error(f"Error fetching user ID by emails {emails}: {e}")
             return None
+
+
+    async def get_user_profile_pic(self, uid: str) -> UserProfileResponse:
+        try:
+            user_record = await asyncio.to_thread(auth.get_user, uid)
```
Comment (vineetshar) [inline app/modules/users/user_service.py:154]: seems very untraditional way of doing it, why are we doing it this way @Yash-pede ?

**Comment by vineetshar** [comment]
Generally looks, good, left one comment needing clarification @Yash-pede 

I was also wondering if firebase will always return a url even if the user has not uploaded their picture for example? Have we tested this scenario?

File: app/modules/users/user_service.py (hunk 3 above)
Comment (Yash-pede) [reply app/modules/users/user_service.py:154]: auth.getuser is not a async function and also to unblock the main thread until it is fetching the user details 
I have showed it @dhirenmathur

Comment (Yash-pede) [reply app/modules/users/user_service.py:154]: Will discuss with him Should i remove it ??

**Review by vineetshar — APPROVED** [review]

**Pull request merged**